art.js

```diff
@@ -42,7 +42,7 @@ function cubesection() {
 }
 
 // Dan's Transparency
-let TRANSPARENCY = 1;
+let TRANSPARENCY = 150;
```

Comment:
Why is it 150 now? As alpha value? Dan answered: To increase the transparency. 1 was opaque.